packages/utxo-lib/src/coinselect/outputs/split.ts

```diff
@@ -23,7 +24,7 @@ export function split(
     feeRate: number,
     options: CoinSelectOptions,
 ): CoinSelectResult {
-    const coinbase = options.coinbase || 100;
+    const coinbase = options.coinbase || MINIMAL_COINBASE_CONFIRMATIONS;
```

Comment:
since you're already modifying it, could you please rename `coinbase` variable? From the name it's not even apparent that it is a number. I'd suggest `minConfCoinbase` to match the argument of `filterCoinbase` :slightly_smiling_face: